InvoiceMaker/InvoiceMaker/Models/WorkDone.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Web;
+
+namespace InvoiceMaker.Models
+{
+    public class WorkDone
+    {
+
+        //private Client _client;
+        //private WorkType _workType;
+
+         public Client Client
+        {
+            get;set;
+        }
+        public WorkType WorkType
+        {
+            get;set;
+        }
+        
+        public WorkDone()
+        {
+
+        }
+        [Required,Column("ClientId")]
+        public int ClientId
```

Comment:
Getter only properties are ignored by Entity Framework, so adding `Required` and `Column` attributes is unnecessary.